src/sentry/api/endpoints/group_similar_issues_embeddings.py

```diff
@@ -49,13 +57,45 @@ def get_stacktrace_string(exception: Mapping[Any, Any], event: GroupEvent) -> st
     return "\n".join(output)
 
 
+class FormattedSimilarIssuesEmbeddingsData(TypedDict, total=False):
+    exception: float
+    message: float
+    shouldBeGrouped: str
+
+
 @region_silo_endpoint
 class GroupSimilarIssuesEmbeddingsEndpoint(GroupEndpoint):
     owner = ApiOwner.ISSUES
     publish_status = {
         "GET": ApiPublishStatus.PRIVATE,
     }
 
+    def get_formatted_results(
+        self, responses: Sequence[Optional[SimilarIssuesEmbeddingsData]], user: User | AnonymousUser
+    ) -> Sequence[Tuple[Mapping[str, Any], Mapping[str, Any]]]:
+        """Format the responses using to be used by the frontend."""
+        group_data = {}
+        for response in responses:
+            if response:
+                formatted_response: FormattedSimilarIssuesEmbeddingsData = {}
+                formatted_response.update({"message": response["message_similarity"]})
```

Comment:
Out of curiosity, is there an advantage to using update instead of combining the items into one dictionary?